docker/serverAndUI/Dockerfile

```diff
@@ -51,4 +51,4 @@ RUN git clone https://github.com/Netflix/conductor.git \
 EXPOSE 5000 8080
 
 CMD ["/app/startup.sh"]
-ENTRYPOINT ["/bin/bash"]
+ENTRYPOINT ["/bin/bash"]
```

Comment:
super nitpicks are welcome.  my intent was to rollback this change completely since i didn't bother to migrate the server-and-ui image to alpine.  i'll clean that up.